src/browser/convert_options.js

```diff
@@ -266,10 +267,6 @@ module.exports = {
             newOptions.preload = options.preload;
         }
 
-        if (options.hasOwnProperty('backgroundThrottling')) {
-            newOptions.backgroundThrottling = options.backgroundThrottling;
-        }
```

Comment:
If it's a part of the base options then this code isn't needed. Adding window options like this actually should be discouraged and only used for options that are used internally, like notifications passing around the notification text [here](https://github.com/HadoukenIO/core/blob/develop/src/browser/convert_options.js#L253).